src/main/java/co/novu/api/workflowoverrides/pojos/WorkflowOverride.java

```diff
@@ -0,0 +1,34 @@
+package co.novu.api.workflowoverrides.pojos;
+
+import co.novu.api.common.PreferenceSettings;
+
+import com.google.gson.annotations.SerializedName;
+
+import lombok.Data;
+
+@Data
+public class WorkflowOverride {
+    @SerializedName("_environmentId")
+    private String _environmentId;
```

Comment:
The essence of @ SerializedName in this context is to remove the underscore from the field name. So it should be **_private String environmentId_** instead